pkg/graph/schema.graphql

```diff
@@ -2436,6 +2436,8 @@ type Mutation {
 
   updateSystemIntakeLinkedCedarSystem(input: UpdateSystemIntakeLinkedCedarSystemInput!): UpdateSystemIntakePayload
 
+  archiveSystemIntake(id: UUID!): UpdateSystemIntakePayload
```

Comment:
not sure what return should be - just `String` with "success" or something? this is ok for now, we can pluck the fields we want via mutation file

Reply:
The payloads aren't used by the FE in a lot of cases from what I've seen, so I don't think it matters much.

Reply:
Yeah, I'm not in love with the `UpdateSystemIntakePayload` type (since it has some weird built-in error type even though GQL already handles that).

Perhaps just returning `SystemIntake!` makes more sense here?

Reply:
updated! thank you